src/libraries/System.Private.CoreLib/src/System/Globalization/OrdinalCasing.Icu.cs

```diff
@@ -12,11 +12,11 @@ namespace System.Globalization
     internal static partial class OrdinalCasing
     {
         // s_noCasingPage means the Unicode page doesn't support any casing and no case translation is needed.
-        private static ushort [] s_noCasingPage = Array.Empty<ushort>();
+        private static readonly ushort[] s_noCasingPage = Array.Empty<ushort>();
```

Comment:
Cases like this could/should be properties instead, yes?

Array.Empty<T>() is already returning a static readonly so caching it in its own here just causes extra lookups and memory waste

Reply:
Yes, this one should just be a property.